modules/library/src/main/scala/zio/elasticsearch/script/Script.scala

```diff
@@ -57,6 +57,6 @@ private[elasticsearch] final case class Script(
 }
 
 object Script {
-  def apply(source: String): Script =
+  def from(source: String): Script =
```

Comment:
Why use `from` instead of `apply`?

Reply:
We agreed earlier that overriding `apply` method is not something we want to be doing as it makes false feeling that Script has only one field.

Reply:
I would consider `source` as a required param. And if we have `Script("source")` then we can consider it as a wrapper on the string. I would use `apply`.

Reply:
Sure. I will change it to `apply`.